cachito/web/api_v1.py

```diff
@@ -74,6 +77,28 @@ def get_requests():
         state_int = RequestStateMapping.__members__[state].value
         query = query.join(RequestState, Request.request_state_id == RequestState.id)
         query = query.filter(RequestState.state == state_int)
+    repo = flask.request.args.get("repo")
+    if repo:
+        query = query.filter(Request.repo == repo)
+    ref = flask.request.args.get("ref")
+    if ref:
+        if not is_request_ref_valid(ref):
+            raise ValidationError(f"{ref} is not a valid ref.")
+        query = query.filter(Request.ref == ref)
+    pkg_managers = flask.request.args.getlist("pkg_manager")
+    if pkg_managers:
```

Comment:
can be `repo`, `ref` and `pkg_managers` defined/used together?

Reply:
@MartinBasti Yes.

Reply:
ah yes, nvm :D